### DISCUSSION TITLE
First Open Source Project Ever! Where do I start?

### DISCUSSION THREAD

**Discussion by rogermayogit** [CodeForVegas/ov-open-transit#2] [Q&A]
If I have never worked on an open source project. Where can I help?  
I know a tiny bit of Basic, Pascal, Fortran, Visual Basic, HTML, CSS, JavaScript, Java, C++, and Python. Most of my work has been done in very simple editors such as notepad, Notepad++, BlueJ, Spider.

**Comment by DanHugoDanHugo** [comment]
Hey there, your pull requests were a good start, though some discussion here would probably be a generally better approach in the early days of the project specifying phase. The original Overview for this project, and all projects, were starters to seed discussion, but we probably need to fill in more blanks in these early days to help with some project onboarding.

It will be difficult to damage anything, experimental entrepreneurialsm is generally encouraged!
